packages/app/src/TeamPage.tsx

```diff
@@ -38,6 +37,29 @@ export default function TeamPage() {
   const hasAllowedAuthMethods =
     team?.allowedAuthMethods != null && team?.allowedAuthMethods.length > 0;
 
+  const sentryDsn = useMemo(() => {
+    if (!team) {
+      return '';
+    }
+
+    try {
+      const ingestorUrl = new URL(process.env.NEXT_PUBLIC_INGESTOR_API_URL);
+
+      const sentryDsnParts = [
+        ingestorUrl.protocol,
+        '//',
+        team.apiKey.replace('-', ''),
+        '@',
+        ingestorUrl.host,
+        ingestorUrl.port ? `:${ingestorUrl.port}` : '',
```

Comment:
For me `ingestorUrl.host` already returns the port so this ended up appending the port twice. We can probably just use `host` on its own since it already includes the port.